proto/regen/ecocredit/orderbook/v1beta1/memory.proto

```diff
@@ -4,7 +4,6 @@ package regen.ecocredit.orderbook.v1beta1;
 
 import "google/protobuf/timestamp.proto";
 import "cosmos/orm/v1alpha1/orm.proto";
-import "regen/ecocredit/marketplace/v1beta1/types.proto";
 
 // BuyOrderSellOrderMatch defines the data the FIFO/price-time-priority matching
 // algorithm used to actually match buy and sell orders.
```

Comment:
how about renaming `BuyOrderSellOrderMatch` to `BuyOrderSellOrderBook`?

Reply:
well it is the actual matches though... would you consider that the order book?

Reply:
Yes, because it includes only the potential matches.